avocado/plugins/yaml_to_mux.py

```diff
@@ -307,12 +382,33 @@ def _log_deprecation_msg(deprecated, current):
         logging.getLogger("avocado.app").warning(msg, deprecated, current)
 
     def run(self, args):
+        # Deprecated filters
+        only = getattr(args, "filter_only", None)
+        if only:
```

Comment:
I know we use custom action elsewhere, but here I thought putting all deprecation code in one class might increase the visibility. I'll reconsider that in the next version...